modules/metering-install-metering-stack.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-installing-metering.adoc
+
+[id="metering-install-metering-stack_{context}"]
+= Installing the Metering stack 
+
+To install the Metering stack create a metering resource and add your configuration details. Review xref:../metering/metering-configuring-metering.adoc#configuring-storage[configuring storage for metering] for more details about all available configuration options. At a minimum you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.
```

Comment:
* Be consistent throughout PR on "Metering stack" or "metering stack"
* We're currently not allowed to xref in modules.